src/main/java/org/junitpioneer/jupiter/ExpectedToFail.java

```diff
@@ -33,6 +33,13 @@
  * of disabled tests over time.
  * </p>
  *
+ * <p>Further, the {@link #onExceptions()} attribute can be used to restrict the extension's behavior
+ * to specific exceptions. I.e. only if the test method end up throwing one of the specified exceptions
+ * will the test be aborted. This can for example be used when the production code temporarily throws
+ * an {@link UnsupportedOperationException} because some feature has not been implemented yet, but the
+ * test method is already implemented and should not be aborted on a failing assertion.
```

Comment:
Should not be aborted?

This doesn't really makes sense. I think this should be "should not fail"?